tasks/add-fbc-contribution/add-fbc-contribution.yaml

```diff
@@ -71,12 +73,24 @@ spec:
         iib_service_account_secret=$(jq -r '.fbc.iibServiceAccountSecret // "iib-service-account"' ${DATA_FILE})
         build_tags=$(jq '.fbc.buildTags // []' ${DATA_FILE})
         add_arches=$(jq '.fbc.addArches // []' ${DATA_FILE})
+        hotfix=$(jq '.fbc.hotfix // "false"' ${DATA_FILE})
         build_timeout_seconds=$(jq -r --arg build_timeout_seconds ${default_build_timeout_seconds} \
             '.fbc.buildTimeoutSeconds // $build_timeout_seconds' ${DATA_FILE})
         target_index=$(jq -r '.fbc.targetIndex' ${DATA_FILE})
         fbc_fragment=$(jq -cr '.components[0].containerImage' ${SNAPSHOT_PATH})
 
-        # saves the target index required to sign
+        if [ "${hotfix}" == "true" ]; then
+          timestamp_format=$(jq -r --arg default_timestamp_format "%s" \
+            '.fbc.timestampFormat // $default_timestamp_format' ${DATA_FILE})
+          timestamp=$(date "+${timestamp_format}")
+          issue_id=$(jq -r '.fbc.issueId' ${DATA_FILE})
```

Comment:
This won't work:
```sh
$ issue_id=$(jq -r '.fbc.issueId' <<< '{"bla": "bla"}')
$ echo $issue_id
null
```
You will need to add `// ""`.

Reply:
I will retry, but I tested the Pipeline dozen of times and it worked.

Reply:
fixed.

Reply:
Yeah, my point was not that the logic should be different, but that without the default, the value of `issue_id` won't be empty even if there is no `issueId` defined.